hnn_core/network.py

```diff
@@ -349,10 +349,15 @@ def add_evoked_drive(self, name, *, mu, sigma, numspikes,
             Standard deviation of event time distribution
         numspikes : int
             Number of spikes at each target cell
-        sync_within_trial : bool
-            If True, the target cells receive each numspike synchronously. By
-            default (False), spike times arriving at each target cell are
-            sampled independently using the Gaussian parameteres (mu, sigma).
+        numsources : int | None
```

Comment:
```suggestion
        n_drive_cells : int | None
```

Wdyt?

Reply:
+1 for `n_drive_cells`. It's more consistent with our current terminology and less ambiguous.

Reply:
+1 for `n_drive_cells`